src/ring_theory/unique_factorization_domain.lean

```diff
@@ -523,6 +523,10 @@ begin
   rw [pow_succ, succ_nsmul, normalized_factors_mul h0 (pow_ne_zero _ h0), ih],
 end
 
+theorem normalized_factors_irreducible_pow {p : α} (hp : irreducible p) (k : ℕ) :
```

Comment:
```suggestion
theorem irreducible.normalized_factors_pow {p : α} (hp : irreducible p) (k : ℕ) :
```
for dot notation? (Is this in a namespare already?)

Reply:
It's in the `unique_factorization_monoid` namespace, so:
```suggestion
theorem _root_.irreducible.normalized_factors_pow {p : α} (hp : irreducible p) (k : ℕ) :
```
(Since `normalized_factors` is a unique enough name that I don't foresee any conflicts.)